ECMA2Yaml/ECMAHelper/ECMA2YamlConverter.cs

```diff
@@ -68,6 +68,7 @@ public class ECMA2YamlConverter
             }
 
             Console.WriteLine("Building loaded files...");
+            Console.WriteLine($"ECMA2YamlRepoConfig:{config}");
```

Comment:
we can just json serialize this object for output

Reply:
Updated